seq2science/rules/configuration_generic.smk

```diff
@@ -42,6 +42,9 @@ docs: https://vanheeringen-lab.github.io/seq2science
 """
 )
 
+include: f"{config['rule_dir']}/explain.smk"
```

Comment:
nice way to minimize repeating code.

we could also add other universal rules at the bottom of this file to clean up the snakefiles. Personally, I would only add the first of these two however.
```
include: f"{config['rule_dir']}/configuration_workflows.smk"
include: f"{config['rule_dir']}/get_fastq.smk"
```

Reply:
This does not seem to work because the order in which these files are read is important :(